app/src/main/java/com/el/yello/presentation/pay/PayResubsNoticeDialog.kt

```diff
@@ -0,0 +1,65 @@
+package com.el.yello.presentation.pay
+
+import android.content.Intent
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.view.View
+import android.view.ViewGroup
+import android.view.WindowManager
+import com.el.yello.R
+import com.el.yello.databinding.FragmentNoticeResubscribeBinding
+import com.example.ui.base.BindingDialogFragment
+import com.example.ui.view.setOnSingleClickListener
+
+class PayResubsNoticeDialog :
+    BindingDialogFragment<FragmentNoticeResubscribeBinding>(R.layout.fragment_notice_resubscribe) {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setNoticeBtnClickListener()
+    }
+
+    override fun onResume() {
+        super.onResume()
+        showDialogFullScreen()
+    }
+
+    private fun showDialogFullScreen() {
+        dialog?.window?.setLayout(
+            ViewGroup.LayoutParams.MATCH_PARENT,
+            ViewGroup.LayoutParams.MATCH_PARENT,
+        )
+        dialog?.window?.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
+        dialog?.window?.clearFlags(WindowManager.LayoutParams.FLAG_DIM_BEHIND)
+    }
+
+    private fun setNoticeBtnClickListener() {
+        binding.btnNoticeQuit.setOnSingleClickListener {
+            dismiss()
+        }
+        binding.btnYelloplusSubscribe.setOnSingleClickListener {
+            val intent = Intent(requireContext(), PayActivity::class.java)
+            startActivity(intent)
+        }
```

Comment:
```suggestion
            Intent(requireContext(), PayActivity::class.java).apply{
                startActivity(this)
            }
        }
```
가 인텐트 객체를 따로 생성하지 않고 진행해서 더 좋을 듯 함둥